docs/csharp/language-reference/keywords/sizeof.md

```diff
@@ -9,25 +9,11 @@ helpviewer_keywords:
 ms.assetid: c548592c-677c-4f40-a4ce-e613f7529141
 ---
 # sizeof (C# Reference)
-Used to obtain the size in bytes for an unmanaged type. Unmanaged types include the built-in types that are listed in the table that follows, and also the following:  
-  
--   Enum types  
-  
--   Pointer types  
-  
--   User-defined structs that do not contain any fields or properties that are reference types  
-  
- The following example shows how to retrieve the size of an `int`:  
-  
-```csharp  
-// Constant value 4:  
-int intSize = sizeof(int);   
-```  
-  
-## Remarks  
- Starting with version 2.0 of C#, applying `sizeof` to built-in types no longer requires that [unsafe](../../../csharp/language-reference/keywords/unsafe.md) mode be used.  
-  
- The `sizeof` operator cannot be overloaded. The values returned by the `sizeof` operator are of type `int`. The following table shows the constant values that are substituted for `sizeof` expressions that have certain built-in types as operands.  
+Used to obtain the size in bytes for an unmanaged type.
+
+Unmanaged types include:
+
+-   The simple types that are listed in the following table:
   
 |Expression|Constant value|  
```

Comment:
Three spaces should be added before each line of the table so that it's indented at the same level as the text of the bulleted list item.